ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandStore.java

```diff
@@ -136,14 +136,14 @@ public CommandId distributeStatement(
         log.debug("Reading prior command records");
 
         List<ConsumerRecord<CommandId, Command>> result = new ArrayList<>();
-        ConsumerRecords<CommandId, Command> records = commandConsumer.poll(30000);
+        ConsumerRecords<CommandId, Command> records = commandConsumer.poll(5000);
```

Comment:
Should we make this configurable rather than hard coding

Reply:
Or at least extract it to a constant as it is used below, too

Reply:
@dguy extracted it to a constant.